src/pycram/datastructures/dataclasses.py

```diff
@@ -478,15 +479,78 @@ class State(ABC):
     pass
 
 
+@dataclass
+class PhysicalBodyState(State):
+    """
+    Dataclass for storing the state of a physical body.
+    """
+    pose: Pose
+    is_translating: bool
+    is_rotating: bool
+    velocity: List[float]
+    acceptable_pose_error: Tuple[float, float] = (0.001, 0.001)
+    acceptable_velocity_error: Tuple[float, float] = (0.001, 0.001)
+
+    def __eq__(self, other: 'PhysicalBodyState'):
```

Comment:
Import from __future__ import annotations and remove the quotation marks

Reply:
done